manifests/globals.pp

```diff
@@ -28,64 +28,23 @@
 #   -> class {'mongodb::server': }
 #   -> class {'mongodb::client': }
 #
-# @param server_package_name
-#   This setting can be used to override the default MongoDB server package name.
-#   If not specified, the module will use whatever package name is the default for your OS distro.
-#
-# @param client_package_name
-#   This setting can be used to specify the name of the client package that should be installed.
-#   If not specified, the module will use whatever service name is the default for your OS distro.
-#
-# @param mongod_service_manage
-#   This setting can be used to override the default management of the mongod service.
-#   By default the module will manage the mongod process.
-# @param service_enable
-#   This setting can be used to specify if the service should be enable at boot
-#
-# @param service_ensure
-#   This setting can be used to specify if the service should be running
-#
-# @param service_name
-#   This setting can be used to override the default MongoDB service name.
-#   If not specified, the module will use whatever service name is the default for your OS distro.
-#
-# @param service_provider
-#   This setting can be used to override the default MongoDB service provider.
-#   If not specified, the module will use whatever service provider is the default for your OS distro.
-#
-# @param service_status
-#   This setting can be used to override the default status check command for your MongoDB service.
-#    If not specified, the module will use whatever service name is the default for your OS distro.
-#
-# @param user
-#   This setting can be used to override the default MongoDB user and owner of the service and related files in the file system.
-#   If not specified, the module will use the default for your OS distro.
-#
-# @param group
-#   This setting can be used to override the default MongoDB user group to be used for related files in the file system.
-#   If not specified, the module will use the default for your OS distro.
-#
-# @param ipv6
-#   This setting is used to configure MongoDB to turn on ipv6 support.
-#   If not specified and ipv6 address is passed to MongoDB bind_ip it will just fail.
-#
-# @param bind_ip
-#   This setting can be used to configure MonogDB process to bind to and listen for connections from applications on this address.
-#   If not specified, the module will use the default for your OS distro.
-#   Note: This value should be passed as an array.
```

Comment:
All of the above is either client or server params. There is no need for them in globals.

Reply:
This is a breaking change.